lib/proxy/plugin_initializer.rb

```diff
@@ -290,6 +301,16 @@ def log_used_settings(settings)
     sorted_keys = settings.keys.map(&:to_s).sort.map(&:to_sym) # ruby 1.8.7 doesn't support sorting of symbols
     to_log = sorted_keys.map {|k| "'%s': %s%s" % [k, settings[k], (default_settings.include?(k) && default_settings[k] == settings[k]) ? " (default)" : ""] }.join(", ")
     logger.debug "'%s' settings: %s" % [plugin.plugin_name, to_log]
+    logger.debug("'%s' ports: 'http': %s, 'https': %s" % [plugin.plugin_name,
+                                                          ::Proxy::PluginGroup::HTTP_ENABLED.include?(settings[:enabled]),
+                                                          ::Proxy::PluginGroup::HTTPS_ENABLED.include?(settings[:enabled])])
+  end
+
+  def log_provider_settings(settings)
+    default_settings = plugin.plugin_default_settings
+    sorted_keys = settings.keys.map(&:to_s).sort.map(&:to_sym) # ruby 1.8.7 doesn't support sorting of symbols
+    to_log = sorted_keys.map {|k| "'%s': %s%s" % [k, settings[k], (default_settings.include?(k) && default_settings[k] == settings[k]) ? " (default)" : ""] }.join(", ")
```

Comment:
It looks like there is a lot of code duplicate between this method and `log_used_settings`, seems like a good candidate to extract the code to some common place